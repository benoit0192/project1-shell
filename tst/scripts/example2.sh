# example2.sh
# an example of valid shell script

# commands can have one argument
uname -a

# or multiple arguments
echo Hello! How are you?
