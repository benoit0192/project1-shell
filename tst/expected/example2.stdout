Minix 10.0.2.15 3.3.0 Minix 3.3.0 (GENERIC) i386
Hello! How are you?
